app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentTest.kt

```diff
@@ -364,6 +366,15 @@ class StateFragmentTest {
     intended(hasComponent(HomeActivity::class.java.name))
   }
 
+  @Test
+  fun testContentCard_forDemoExploration_withCustomOppiaTags_displaysParsedHtml() {
+    ActivityScenario.launch(ContentCardTestActivity::class.java).use {
```

Comment:
When a separate activity is created for testing content card why we are not having separate test class for it?.
Is it supposed to be in StateFragmentTest?

Reply:
As it is a part of StateFragment it was suggested by @BenHenning  via @rt4914 to keep test case in in StateFragmentTest.